README.md

```diff
@@ -280,8 +287,13 @@ Still, helping the solver is usually a good idea, so let's give ̀`Ipopt` a star
 The initial guess that we can provide are those for the states (`x_init`, here *q* and *qdot*) and for the controls (`u_init`, here *tau*). 
 So let's define both of them quickly
 ```python
-x_init = InitialGuess([0, 0, 0, 0])
+x_init = InitialGuess([0, 0, 0, 0]) 
 u_init = InitialGuess([0, 0])
+
+Each element declared in x_init is the value of the degree of freedom in the bio.Mod file : 
+for example x_init = InitialGuess([q1, q2, qdot1, qdot2]) with q1,q2 the positions and qdot1,qdot2 the velocities of q1 and q2 
```

Comment:
For?

please put codes and keywords inside the "accent grave" brackets (as in à without the a under :P) and q1 and so on inside star to make them italic